server/src/tenant/metadata/metadata.datasource.ts

```diff
@@ -9,7 +9,7 @@ config();
 const configService = new ConfigService();
 
 export const typeORMMetadataModuleOptions: TypeOrmModuleOptions = {
-  url: configService.get<string>('PG_DATABASE_URL')!,
+  url: configService.get<string>('PG_DATABASE_URL'),
```

Comment:
@magrinj @Weiko we should actually use environmentService here!

Reply:
@charlesBochet Actually we can't use it here as we're outside Nest.JS context